### PR TITLE
fix pk can't set value when pk is point value or rename

### PR DESCRIPTION
**Pull request by biless**

### CODE REVIEW

File: helpers.go [hunk 1]
```diff
@@ -196,25 +195,43 @@ func isArrayValueZero(v reflect.Value) bool {
...
 	case reflect.Uint64:
-		v = uint64(id)
+		temp := uint64(id)
+		v = &temp
 	case reflect.Uint:
-		v = uint(id)
+		temp := uint(id)
+		v = &temp
+	}
+	fmt.Println(v)
```
Comment (lunny) [inline helpers.go]: unremoved trace

**Comment by lunny** [comment]
Hi, could you rebase to master since I have added mysql & postgres  tests on circle CI.

**Comment by lunny** [comment]
will fix #586

**Comment by biless** [comment]
这个错误似乎我的无关吧

**Comment by lunny** [comment]
```
--- FAIL: TestInsertOneIfPkIsPoint (0.02s)
	Error Trace:	session_insert_test.go:44
	Error:      	Received unexpected error:
	            	pq: null value in column "id" violates not-null constraint
```
Because before this PR, pointer to primary key is not supported, so this PR should fix this to avoid insert nil to an autoncriment column.

**Comment by lunny** [comment]
Thanks!

**Pull request merged**